project2/styles.css

```
@@ -0,0 +1,121 @@
+* {
+    box-sizing: border-box;
+    transition: all 1s;
+}
+
+body {
+    background-color: #111111;
+    color: e0e0e0;
```

This is missing the `#` at the front of this hex code for your text color:
```suggestion
    color: #e0e0e0;
```